src/lib.rs

```diff
@@ -341,11 +403,11 @@ pub async fn compare_attestations(
     } else {
         info!(
             "Number of nPOI submitted for block {}: {:#?}\n{}: {:#?}",
-            attestation_block, remote_attestations, "Local attestation", local_attestation
+            attestation_block, remote_attestations, "Local supported POI", &local_attestation.npoi
```

Comment:
Is "supported" the right word here? It makes me think that it's "supported" by something external, but it's just the local npoi 🤔